src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  background-color: $white-color;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
```

Comment:
The 'box-sizing: border-box;' property is declared twice. It's redundant to have it on both line 2 and line 11. Consider removing one of them.